component/src/main/java/io/siddhi/extension/io/file/listeners/FileSystemListener.java

```diff
@@ -200,43 +171,33 @@ public boolean onMessage(RemoteFileSystemBaseMessage remoteFileSystemBaseEvent)
         }
     }
 
-    @Override
-    public void onError(Throwable throwable) {
-    }
-
-    @Override
-    public void done() {
-    }
-
-    static class FileServerExecutor implements Runnable {
-        ServerConnector fileServerConnector = null;
-        CarbonCallback carbonCallback = null;
-        CarbonMessage carbonMessage = null;
-        String fileURI = null;
-
-        FileServerExecutor(CarbonMessage carbonMessage, CarbonCallback carbonCallback,
-                           ServerConnector fileServerConnector, String fileURI) {
-            this.fileURI = fileURI;
-            this.fileServerConnector = fileServerConnector;
-            this.carbonCallback = carbonCallback;
-            this.carbonMessage = carbonMessage;
-        }
-
-        @Override
-        public void run() {
+    public void runProgram(VFSClientConnector vfsClientConnector,
```

Comment:
@ajanthy shall we change the name of this method to express what this method does?
eg: startFileProcessing

Reply:
fixed in 47cc199b8188f3701d228a031aeb7a33f02bc856